actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -129,22 +137,18 @@ def distance_of_time_in_words(from_time, to_time = 0, options = {})
             # 60 days up to 365 days
             when 86400...525600   then locale.t :x_months,       :count => (distance_in_minutes.to_f / 43200.0).round
             else
-              if from_time.acts_like?(:time) && to_time.acts_like?(:time)
-                fyear = from_time.year
-                fyear += 1 if from_time.month >= 3
-                tyear = to_time.year
-                tyear -= 1 if to_time.month < 3
-                leap_years = (fyear > tyear) ? 0 : (fyear..tyear).count{|x| Date.leap?(x)}
-                minute_offset_for_leap_year = leap_years * 1440
-                # Discount the leap year days when calculating year distance.
-                # e.g. if there are 20 leap year days between 2 dates having the same day
-                # and month then the based on 365 days calculation
-                # the distance in years will come out to over 80 years when in written
-                # English it would read better as about 80 years.
-                minutes_with_offset = distance_in_minutes - minute_offset_for_leap_year
-              else
-                minutes_with_offset = distance_in_minutes
-              end
+              fyear = from_time.year
```

Comment:
Can we be a bit more explicit with these? It took me a minute to figure out what `fyear` meant.